src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApAreaEntity.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.OneToMany
+import javax.persistence.Table
+
+@Entity
+@Table(name = "ap_area")
+data class ApAreaEntity(
+  @Id
```

Comment:
Can we make the IDs UUIDs please?

Reply:
Sure, just for ApArea or also for LocalAuthority and ProbationRegion?

Reply:
All of them please - LocalAuthority and ProbationRegion will also need seperate `identifier` fields for the internal IDs

Reply:
can do - `identifier` would be a string?